package_metrics/datadog_utils.py

```diff
@@ -13,6 +13,19 @@
 )
 
 
+def send_stats_to_datadog(stats, package_manager):
+    """
+    :param stats: key-value pairs representing outdated counts
+    Example: {'major': 3, 'minor': 2, ...})
+    :param package_manager: package manager used to collect statistics
+    """
+    metric_env = get_metric_name_for_package_manager(package_manager)
+    metric_prefix = f"commcare.static_analysis.dependency.{metric_env}"
```

Comment:
Are we also adding the name of repo in the metrics?
For eg- How will we differentiate that the metric came from Vellum or HQ?